app/assets/stylesheets/chart-styles.css.scss

```diff
@@ -62,4 +62,20 @@
     stroke: black;
   }
 
+  /*JESSIE EDITS*/
+g.data-points circle {
+  opacity: 0.3;
+  fill: grey;}
+
+g.x.axis text {
```

Comment:
Line should be indented 2 spaces, but was indented 0 spaces<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3